app.tsx

```diff
@@ -9,20 +9,61 @@
  */
 
 import React from 'react';
-import { SafeAreaView, Text, useColorScheme } from 'react-native';
+import { SafeAreaView, Text, useColorScheme, TextInput, View, Button } from 'react-native';
 
 import { Colors } from 'react-native/Libraries/NewAppScreen';
 
 const App = () => {
   const isDarkMode = useColorScheme() === 'dark';
+  const [emailValue, setEmailValue] = React.useState('');
+  const [passwordValue, setPasswordValue] = React.useState('');
+  const [error, setError] = React.useState('');
 
   const backgroundStyle = {
-    backgroundColor: isDarkMode ? Colors.darker : Colors.lighter,
+    backgroundColor: isDarkMode ? Colors.darker : Colors.white,
   };
 
   return (
     <SafeAreaView style={backgroundStyle}>
-      <Text>Hello world!</Text>
+      <View>
+        <Text>E-mail</Text>
+        <TextInput
+          placeholder='ex:joao.silva@gmail.com'
+          onChangeText={(text) => {
+            setEmailValue(text);
+          }}
+        ></TextInput>
```

Comment:
hmm só duas coisas aqui:
1) como esse componente não tem `children`, vc pode usá-lo assim:
```suggestion
        <TextInput
          placeholder='ex:joao.silva@gmail.com'
          onChangeText={(text) => {
            setEmailValue(text);
          }}
        />
```
2. Esse é mais uma pergunta kkk O `emailValue` (do `useState`) não está sendo usado. pra que ele serve?

Reply:
O `emailValue` é usado na verificação quando pressiona o botão

Reply:
hmmm boa. 
então nesse caso, como o TextInput está [uncontrolled](https://reactjs.org/docs/uncontrolled-components.html), você pode usar `useRef` ao invés de `useState`. o que acha? (qualquer coisa chama no slack 👍 )